R/validate_table_struct.R

```diff
@@ -1,18 +1,18 @@
-#' Find degenerate (sub)structures within a table (Experimental)
-#' @param tt `TableTree`
+#' Find degenerate (sub)structures within a table
 #'
-#' This function returns a list with the row-paths to all
-#' structural subtables which contain no data rows (
-#' even if they have associated content rows).
+#' @description `r lifecycle::badge("experimental")`
 #'
-#' @return a list of character vectors representing the
-#' row paths, if any, to degenerate substructures within
-#' the table.
+#' This function returns a list with the row-paths to all structural subtables which contain no data rows (even if
+#' they have associated content rows).
+#'
+#' @param tt (`TableTree`)\cr a `TableTree` object.
+#'
+#' @return A list of character vectors representing the row paths, if any, to degenerate substructures within the table.
 #'
-#' @export
 #' @examples
 #' find_degen_struct(rtable("hi"))
 #'
+#' @export
```

Comment:
are all these validation functions in the same documentation file? or otherwise connected between each others? I think otherwise users (and myself too) will not find useful connected functions

Reply:
I have added a family to link these functions :)